web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v407/migrate-default.sql

```diff
@@ -1,2 +1,6 @@
 UPDATE Settings SET value='4.0.7' WHERE name='system/platform/version';
 UPDATE Settings SET value='SNAPSHOT' WHERE name='system/platform/subVersion';
+
+UPDATE Settings set position = 7213 WHERE name = 'system/inspire/remotevalidation/nodeid';
+UPDATE Settings set position = 7214 WHERE name = 'system/inspire/remotevalidation/apikey';
```

Comment:
Why this two settings are part of the 4.0.7 migration?

Reply:
This is to update the position of these elements and display the new URL, below the default one.